src/main/java/org/logstash/beats/Server.java

```diff
@@ -34,25 +31,22 @@
 
     private final int clientInactivityTimeoutSeconds;
 
-    public Server(String host, int p) {
-        this(host, p, DEFAULT_CLIENT_TIMEOUT_SECONDS, Runtime.getRuntime().availableProcessors() * 4);
-    }
-
-    public Server(String host, int p, int timeout, int threadCount) {
+   public Server(String host, int p, int timeout, int threadCount) {
+//       ResourceLeakDetector.setLevel(ResourceLeakDetector.Level.PARANOID);
```

Comment:
@robbavey I guess this should be cleaned up before merging :)

Reply:
👍